lua/easychat/modules/client/steam_emojis.lua

```diff
@@ -103,7 +85,7 @@ local function get_steam_emote(name)
 			return fail("ending")
 		end
 
-		local start2, ending2 = data:find([["]], ending + 64, true)
+		local start2 = data:find([["]], ending + 64, true)
```

Comment:
I usually put discards when a function returns varargs because its better for further readings, helps you know what returns what